### PR TITLE
SCC How Do I?

### PR DESCRIPTION
**Pull request by pweil-**
Add a section for some common questions

/cc @adellape - will rebase, just wanted to get this created.

### CODE REVIEW

**Comment by adellape** [comment]
@pweil- https://github.com/openshift/openshift-docs/pull/524 has merged.

**Comment by adellape** [comment]
@pweil- Since this new content is procedure/task-focused, could it work as its own Admin Guide topic? Which the Arch topic could link to at its end.

**Comment by pweil-** [comment]
> Since this new content is procedure/task-focused, could it work as its own Admin Guide topic? Which the Arch topic could link to at its end.

Probably a good idea

**Comment by adellape** [comment]
Possible titles:

"Managing Security Context Constraints"
"Managing Security" (maybe too broad)

**Comment by pweil-** [comment]
Updated to be a new admin section

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+link:../architecture/additional_concepts/authorization.html#security-context-constraints[security context constraints]
+in your instance via the `oc` command as a normal API objects.
+
+When you perform node management operations, the CLI interacts with
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-specification[node
+objects] that are representations of actual node hosts. The
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master]
+uses the information from node objects to validate nodes with
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[health
+checks].
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: Think this is copy/paste leftover.

File: admin_guide/manage_scc.adoc [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+== Creating New Security Context Constraints
+
+Creating a new SCC is accomplished with the `oc create` command.  First, define the SCC in a json
+or yaml file and then run `oc create` passing the file.
+
+[options="nowrap"]
+----
+{
+  "kind": "SecurityContextConstraints",
+  "apiVersion":"v1beta3",
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: We'll need to convert this to v1.

File: admin_guide/manage_scc.adoc [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  },
+  "allowPrivilegedContainer": true,
+  "runAsUser": {
+    "type": "RunAsAny"
+  },
+  "seLinuxContext": {
+    "type": "RunAsAny"
+  },
+  "groups": ["system:serviceaccounts"]
+}
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: Are there any particular fields in this spec we should use callouts on for descriptions? Although maybe that's more appropriate for the `oc export` output in the Arch topic.

Comment (adellape) [inline admin_guide/manage_scc.adoc]: Also, if it's all the same, think we're trying to use YAML for examples in docs since it's friendlier to read.

Comment (pweil-) [inline admin_guide/manage_scc.adoc]: notes added to the export command

File: admin_guide/manage_scc.adoc (hunk 2 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: converted

File: admin_guide/manage_scc.adoc (hunk 1 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: removed

File: admin_guide/manage_scc.adoc [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+scc-admin    true      []        false     RunAsAny    RunAsAny
+----
+
+== Deleting New Security Context Constraints
+
+[options="nowrap"]
+----
+$ oc delete scc scc-admin
+securitycontextconstraints/scc-admin
+----
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: I tried seeing what happened when I deleted the default SCCs. It seemed that if I deleted one or the other, and restarted openshift, they did not regenerate. But if I delete both, then restarted, they both came back. Is that accurate / worth mentioning as a warning here?

File: admin_guide/manage_scc.adoc (hunk 3 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: updated to yaml

File: admin_guide/manage_scc.adoc (hunk 4 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: That is correct, added as NOTE:

Comment (adellape) [inline admin_guide/manage_scc.adoc]: Awesome, thanks.

**Comment by adellape** [comment]
@pweil- Latest LGTM. I can merge when the build passes if you're set, and I'll probably just some minor clean-up I'll do in a follow-up.

**Comment by pweil-** [comment]
All set, thanks for the review.

**Pull request merged**